treeherder/etl/management/commands/ingest.py

```diff
@@ -306,13 +310,69 @@ def ingest_git_pushes(project, dry_run=False):
             logger.warning("{} does not match {}".format(revision, th_pushes[index]["revision"]))
 
 
+def _handle(self, *args, **options):
+    project = options['project']
+    changeset = options['changeset']
+
+    if not options['last_n_pushes'] and not changeset:
+        raise CommandError('must specify --last-n-pushes or a positional '
+                           'changeset argument')
+
+    # get reference to repo
+    repo = Repository.objects.get(name=project, active_status='active')
+
+    if options['last_n_pushes']:
```

Comment:
Let's assume this function *only* handles last N pushes so remove the else clause.